nncf/torch/graph/operator_metatypes.py

```diff
@@ -918,6 +919,14 @@ class PTEmbeddingMetatype(PTOperatorMetatype):
     weight_port_ids = [1]
 
 
+@FX_OPERATOR_METATYPES.register()
+class FXEmbeddingMetatype(OperatorMetatype):
```

Comment:
```suggestion
class PTAtenEmbeddingMetatype(OperatorMetatype):
```
What do you think about this naming?

Reply:
Looks more suitable, I will change it!

Reply:
Done!